filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
+    it = iter(params)
+    return any(it) and not any(it)
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'only_one_true': only_one_true
```

Comment:
I really don't think we need the extra function. To ensure the use case you propose, this could be:

    { 'only_one_true':  lambda *args: args.count(True) == 1 }

If we insist in a function, this would look like:

   def only_one_true(*args):
        return args.count(True) == 1